index.js

```diff
@@ -36,6 +36,15 @@
             context.map.center = new L.LatLng(center[1], center[0]);
             context.map.zoom = center[2];
         },
+        bounds: function(context, b) {
+            // TileJson order is lng lat lng lat
+            // left, bottom, right, top
+            var left = b[0], bottom = b[1], right = b[2], top = b[3];
+            context.bounds = L.latLngBounds([bottom, left], [top, right]);
```

Comment:
You are, of course, correct. 

I'm also unsure if Proj4Leaflet supports that, so I left it at that for now.